base/dict.jl

```diff
@@ -493,6 +559,12 @@ julia> get(d, "a", 3)
 julia> get(d, "c", 3)
 3
 ```
+
+Using `get()` is the same as using `d[x]` syntax when the key exists in the collection. However, when the key doesn't exist, `get` returns the default value while `d[x]` throws an error.
```

Comment:
I don't really understand this part. Does it say anything that the first sentence doesn't?

Reply:
`d[x]` returns an error but `get` gives the default value.